### DISCUSSION TITLE
Firefix browser compatbility

### DISCUSSION THREAD

**Discussion by DagCatoSkarvik** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#22] [Q&A] [answered]
Has the site been tested on Firefox? I was able to log in via Edge and Chrome, but not Firefox.

**Comment by fjallemark** [comment, marked as answer]
Yes, users can log in using Firefox. Check you cookie permission settings in Firefox. The Module Registry requires cookies to be stored in your browser.
